### DISCUSSION TITLE
FormLabel custom style component sizes not working

### DISCUSSION THREAD

**Discussion by mahmudinm** [chakra-ui/chakra-ui#5853] [Q&A]
<img width="1094" alt="Screen Shot 2022-04-11 at 14 25 08" src="https://user-images.githubusercontent.com/21211150/162686209-e27c3b0f-0d39-4ad7-843c-43042a9667b6.png">

<img width="661" alt="image" src="https://user-images.githubusercontent.com/21211150/162686310-290a39ae-4476-41a3-90aa-19cc5af013ca.png">

im trying to change size on md size but is not working

**Comment by chr-ge** [comment]
Should work fine if you specify the size. [Here is a working example.](https://codesandbox.io/s/agitated-sea-3nnm3k?file=/src/index.tsx)
